Contributing.md

```diff
@@ -29,7 +26,7 @@ Following this guide and formatting your code as detailed will likely get your p
 This project uses clang-format (stable branch) to check for common style issues. In case of conflicts between this guide and clang-format rules, the latter should be followed instead of this guide.
 
 
-### Checking and fixing formatting issues
+## <a name="intro-formatting-issues"></a>Checking and fixing formatting issues
```

Comment:
Changed them all to sentence-case, since when the guidelines are made into referenceable items, they won't be title-cased.